crowbar_framework/app/models/network_service.rb

```diff
@@ -413,8 +412,6 @@ def build_net_info(network, name, db = nil)
     db["network"].each { |k,v|
       net_info[k] = v unless v.nil?
     }
-    net_info["usage"]= network
-    net_info["node"] = name
```

Comment:
Ah, got it: what you need is the parameter to this call. Yes, better to add it back later on if needed, imho. Not a big fan of keeping unused parameters.